src/CaseHeader/CaseHeader.tsx

```diff
@@ -94,4 +95,4 @@ class CaseHeader extends Component {
 export default compose(
 	withStyles(styles),
 	withWidth()
-)(CaseHeader);
+)(translate(['cases'], { wait: true })(CaseHeader));
```

Comment:
This is a good example for the use of decorators. I would love to be able to do this (but it would need more work):

```
@translate(['cases'], {wait: true})
@withStyles(styles)
@withWidth()
export default class CaseHeader extends Component<CaseHeaderProps> {
...
}
```

Reply:
That would be really nice!